lib/govuk_app_config/govuk_healthcheck/checkup.rb

```diff
@@ -0,0 +1,26 @@
+module GovukHealthcheck
+  OK = "ok".freeze
+  CRITICAL = "critical".freeze
+
+  class Checkup
+    # @param checks [Array] Array of objects/classes that respond to `run`
+    def initialize(checks)
+      @checks = checks
+    end
+
+    def as_json
+      {
+        status: OK,
```

Comment:
https://github.com/alphagov/email-alert-api/pull/234 seems to imply that the overall status is important too, do you know what the difference is to Icinga?